harness/determined/keras/_tf_keras_context.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext):
         self._fit_workers = 1
         self._fit_use_multiprocessing = False
         self._fit_max_queue_size = 10
+        self._fit_shuffle = True
```

Comment:
praise: thank you for matching the tf.keras fit()0 default

Reply:
Yeah it's about time we supported shuffle haha